02.md

```diff
@@ -42,7 +46,31 @@ Welcome Events use [NIP-59](https://github.com/nostr-protocol/nips/blob/master/5
 }
 ```
 
-**Important**: This event is NOT signed (`"sig"` field omitted) to prevent accidental public publishing if it leaks.
+### Gift Wrap (Relay-Facing Envelope)
+
+Wrap the rumor following [NIP-59](https://github.com/nostr-protocol/nips/blob/master/59.md):
+
+1. Encrypt the rumor into a `kind: 13` seal signed by the inviter’s identity key.
+2. Encrypt the seal into a `kind: 1059` gift wrap signed by a single-use wrapper key. This is the only event relays see.
+
+```json
+{
+  "id": "wrap123...",
+  "kind": 1059,
+  "created_at": 1693876601,
+  "pubkey": "11aa22bb33cc44dd55ee66ff77889900aabbccddeeff00112233445566778899",
+  "content": "Encrypted kind-13 seal containing the rumor",
+  "tags": [
+    ["p", "invited_member_pubkey"]
+  ],
+  "sig": "6b9450..."
+}
+```
+
+**Signing requirements**:
+- The rumor remains unsigned to preserve NIP-59 deniability.
+- The `kind: 13` seal MUST be signed by the inviter (authenticating the Welcome).
+- The `kind: 1059` gift wrap MUST be signed by its wrapper key so relays will accept and deliver it.
```

Comment:
TBH this all seems redundant.

- NIP-59 is already linked in line 28
- The Gift-Wrap NIP already explains the seal / wrap logic